docs/extras/advanced-variables/variable-defaults.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Variable Defaults
+sidebar_position: 2
+---
+
+Variable defaults are the values you provide in code which tell DevCycle SDKs what value to assign to a Variable in
```

Comment:
This may be a question for the future, but is the a quick visualization we could create here to explain when defaults would be used in a simple way?

Reply:
I agree a diagram could be helpful, but I can't really picture what it should look like.

Reply:
@andrewdmaclean I wont have time to make a diagram right now, so I think Im going to merge this without for now and maybe we can add one in the future